Code/Framework/AzFramework/AzFramework/Components/TransformComponent.h

```diff
@@ -190,5 +197,11 @@ namespace AzFramework
         bool m_isStatic = false; ///< If true, the transform is static and doesn't move while entity is active.
         /// Behavior for this entity's transform when its parent's transform changes.
         AZ::OnParentChangedBehavior m_onParentChangedBehavior = AZ::OnParentChangedBehavior::Update;
+
+        // carbonated begin aoreshko game_specific_1
```

Comment:
Please remove aoreshko here and the other places, we drop names from engine patches.